lib/open_api_spex/test/assertions.ex

```diff
@@ -10,26 +10,31 @@ defmodule OpenApiSpex.Test.Assertions do
   @doc """
   Asserts that `value` conforms to the schema with title `schema_title` in `api_spec`.
   """
-  @spec assert_schema(map, String.t, OpenApi.t) :: map | no_return
+  @spec assert_schema(map, String.t(), OpenApi.t()) :: map | no_return
+  @deprecated "Use OpenApiSpex.TestAssertions.assert_schema/3 instead"
   def assert_schema(value = %{}, schema_title, api_spec = %OpenApi{}) do
```

Comment:
Why not just delegate to `OpenApiSpex.TestAssertions.assert_schema/3`?

Reply:
The behavior is different than `OpenApiSpex.TestAssertions.assert_schema/3`. Also, this model is being deprecated, so it doesn't make sense to make any other changes to it.